cli/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cloudquery/cloudquery/cli
 go 1.19
 
 require (
-	github.com/cloudquery/plugin-sdk v1.42.0
+	github.com/cloudquery/plugin-sdk v1.42.1-0.20230307170513-f5cd8633c7a0
```

Comment:
This points to a commit sha https://github.com/cloudquery/plugin-sdk/pull/726/commits/f5cd8633c7a0cfb3e8f9c77bdea67df4d411c150.

We should use the official version before merging